src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -65,28 +65,26 @@ class NotificationService(
         val newShiftNotifications = allPrisons
                 .flatMap { prison ->
                     csrClient.getShiftNotifications(prison.csrPlanUnit, prison.region)
-                            .filter { !checkIfNotificationsExist(it.quantumId, it.shiftDate, it.shiftType) }
-                            .map {
-                                it.actionType = ShiftActionType.ADD.value
-                                it
-                            }
+                            .filter { !checkIfNotificationsExist(it.quantumId, it.shiftDate, it.shiftType, it.shiftModified) }
+
```

Comment:
This changes ALL the notification types to ADD, meaning we are sending the wrong description to users. I misunderstood what it was doing on the last PR. We only want to update the value for EDIT types where is there is no previous ADD type. I'm not convinced its important enough to replicate in the new code, and I think it became irrelevant because the old code was only saving ADD types but the stored procedure had subsequently been changed to send ADD/EDIT/DELETE.